model/src/test/test_infection_seeding_method.py

```diff
@@ -0,0 +1,79 @@
+# numpydoc ignore=GL08
+import numpy as np
+import numpy.testing as testing
+from pyrenew.deterministic import DeterministicVariable
+from pyrenew.latent import (
+    SeedInfectionsExponential,
+    SeedInfectionsFromVec,
+    SeedInfectionsZeroPad,
+)
+
+
+def test_seed_infections_exponential():
+    """Check that the SeedInfectionsExponential class generates the correct number of infections at each time point."""
```

Comment:
have a test that an error is raised if `I_pre_seed_RV` is not a scalar as required by `SeedInfectionsExponential`?

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/124/commits/50b91e4f54585f290888b686c6d9f53478cf11f8